run-genny

```diff
@@ -124,6 +125,7 @@ if [[ ! -d "${GENNY_REPO_ROOT}/${_genny_venv}" || ! -e "${GENNY_REPO_ROOT}/${_ge
     source "${GENNY_REPO_ROOT}/${_genny_venv}/bin/activate"
 
     # Install Genny requirements
+    _pip install "numpy==1.22.3"  # For some reason pip doesn't pick this up in requirements.txt?
```

Comment:
you sure? This worked in a clean /old env for me:

```
$ genny_venv/bin/pip show numpy
Name: numpy
Version: 1.22.3
Summary: NumPy is the fundamental package for array computing with Python.
Home-page: https://www.numpy.org
Author: Travis E. Oliphant et al.
Author-email: None
License: BSD
Location: /home/jim/git/genny-trees/master/genny_venv/lib/python3.9/site-packages
Requires: 
Required-by: numexpr

```

Reply:
I fought this for over an hour and eventually kinda threw my hands up. What worked on macOS didn't work on Ubuntu and vice-versa, and installing on macOS makes it compile numpy from source in many circumstances. I lost my will to make this better.